docs/tutorials/tutorial_sy.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: page
+---
+
+# Tutorial: Change the due-date of a task
+
+In this tutorial, you change the due-date of a task in the service.
+
+Expect this tutorial to take about 15 minutes to complete.
+
+## Before you start
+
+1. Make sure you've completed the [Before you start a tutorial](../before-you-start-a-tutorial.md) topic on the development system you'll use for the tutorial.
+
+2. Make sure that a task already exists in the service or follow the [Add a new task tutorial](add-a-new-task.md). You'll need the `id` of the [`task`](../api/task.md) resource that you'll be changing.
+
+## Change a task's due-date
+
+Changing a task's due-date requires that you `PATCH` the `due_date` property of a [`task`](../api/task.md) resource.
+
+To change the task's due-date:
+
+1. Make sure your local service is running, or start it by using this command, if it's not.
+
+    ```shell
+    cd <your-github-workspace>/to-do-service/api
+    json-server -w to-do-db-source.json
+    ```
+
+2. Run this command to make a `PATCH` request. Substitute `<date>` with the desired due-date in [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601) format, and `<id>` with the `id` of the task that should be changed.
```

Comment:
```suggestion
2. In another command-line window, run this command to make a `PATCH` request. Substitute `<date>` with the new due-date in [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601) format, and `<id>` with the `id` of the task that should be changed.
```
Because there are two things to substitute, you could also list them in a bullet list under the ordered list item by intending the list entries. This would make each property that needs to be updated more visible.

Reply:
Done!